src/pages/SetupPlot.vue

```diff
@@ -249,10 +250,9 @@ export default defineComponent({
       })
       if (this.defaultPath != this.plotDirectory)
         await native.createDir(this.plotDirectory)
-      const farmerIdentity = await startFarming(this.plotDirectory)
-      // TODO: find a way to store and retrieve the public key from client.ts.
-      LocalStorage.set("farmerPublicKey", farmerIdentity.publicKey)
-      LocalStorage.set("mnemonic", farmerIdentity.mnemonic)
+      const { publicKey, mnemonic } = await startFarming(this.plotDirectory)
+      if(publicKey && mnemonic)
+        await this.client.init(publicKey, mnemonic) 
```

Comment:
maybe we can display the mnemonic here directly in a modal, without storing it. 
This will resolve the potential security vulnerability we have (storing a piece of sensitive information).